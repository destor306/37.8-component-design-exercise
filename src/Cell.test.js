import React from 'react';
import { render} from '@testing-library/react';
import Cell from './Cell';


test('it render without crashing', ()=>{
    render(<Cell />);
})


// snapshot


test("it should matches snapshot", ()=>{
    const {asFragment} = render(<Cell />);
    expect(asFragment()).toMatchSnapshot();
})