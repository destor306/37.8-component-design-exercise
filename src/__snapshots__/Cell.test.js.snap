// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should matches snapshot 1`] = `<DocumentFragment />`;
